src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant/ClientCredentialsOptions.cs

```diff
@@ -32,12 +32,17 @@ public sealed class ClientCredentialsOptions
     /// </summary>
     public string Scope
     {
-        get => string.Join(" ", this.Scopes);
+        get => this.Scopes == null ? string.Empty : string.Join(" ", this.Scopes);
         set => this.Scopes = value.Split(SpaceCharacterArray, StringSplitOptions.RemoveEmptyEntries);
     }
 
     /// <summary>
     /// A time buffer during which an access token is deemed expired prior to its actual expiration time.
     /// </summary>
     public TimeSpan CacheLifetimeBuffer { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
Is it something that we want to be able to customize but have a default value? Right now, 1 minute seems very low

Reply:
Yes so it is definitely customizable but we have this default "1 minute" value that came from Duende's original implementation.

So if a JWT token has a TTL of 15 minutes, we'll keep it 14 minutes in the cache (15 minutes - CacheLifetimeBuffer)